pkg/metrics/metrics.go

```diff
@@ -85,6 +85,13 @@ var (
 		"source",
 		"reason",
 	)
+	filterFlows = defineMetric(
+		"filtered_flows_total",
+		"Number of filtered flows",
+		TypeCounter,
+		"source",
+		"reason",
+	)
```

Comment:
Do we need to update the dashboards ?

Maybe we can manage that in https://github.com/netobserv/network-observability-operator/pull/607 or create a followup in Jira

Reply:
not sure if we need to update the dashboard or we just add it to the query list not sure @jotak but that is correct we can either added to 607 PR on a following PR

Reply:
let's do a follow-up JIRA then; I would say the goal for NETOBSERV-1478 is reached already

For users who enable filtering, I guess it can be interesting to know the ratio of filtered in/filtered out flows indeed, so yeah why not adding that to the health dashboard